lib/nrf_modem_lib/lte_net_if/lte_net_if.c

```diff
@@ -25,6 +25,12 @@ static void connection_timeout_schedule(void);
 /* Delayable work used to handle LTE connection timeouts. */
 static K_WORK_DELAYABLE_DEFINE(connection_timeout_work, connection_timeout_work_fn);
 
+/* Forward declaration of modem fault handler task. */
+static void modem_fault_handler_task(struct k_work *work);
```

Comment:
```suggestion
static void modem_fault_handler_work_fn(struct k_work *work);
```
Could the naming be aligned with the already existing work item/function?

Reply:
Done